bin/build.rb

```diff
@@ -3,9 +3,19 @@
 $LOAD_PATH << File.expand_path("../lib", __dir__)
 
 require 'manageiq/rpm_build'
+require 'optimist'
+
+opts = Optimist.options do
+  opt :build_type, "nightly or release", :type => :string, :default => "nightly"
+  opt :git_ref,    "Git ref to use (default: git_ref specified in options.yml)", :type => :string
+end
+
+build_type = opts[:build_type]
+git_ref    = opts[:git_ref]
+Optimist.die "git_ref must be set for release build" if build_type == "release" && git_ref.nil?
```

Comment:
If you specify the `git_ref` in the options yml, and then set `--build_type=release` then this will fail.

I think the line above needs to be something like `opts[:git_ref] || config_options[:git_ref]`, or perhaps just drop this check altogether.

Reply:
That's true... it doesn't make sense to check if default git ref is in options.yml file. I'll drop this.